code/game/gamemodes/nations/nations.dm

```diff
@@ -16,9 +16,8 @@ datum/game_mode/nations
 		set_ai()
 		assign_leaders()
 //		remove_access()
-		for(var/mob/M in GLOB.player_list)
-			if(!istype(M,/mob/new_player))
-				M << sound('sound/effects/purge_siren.ogg')
+		set_security_level("epsilon")
```

Comment:
have this call `del(world)`.

Reply:
Should make some other even more nasty security level that's really just a wrapper for a force-restart of the server with no OOC/chat notification. Literally just ending reality.

Reply:
![](https://i.imgur.com/7nBynaN.png)

Reply:
See, that's the real problem. I think people are actually enjoying the game.